### PR TITLE
fix: fail when none of the source or customizations folder is given

### PR DESCRIPTION
**Pull request by kmehant**
Signed-off-by: Mehant Kammakomati <kmehant@gmail.com>

### CODE REVIEW

File: lib/planner.go [hunk 1]
```diff
@@ -28,6 +28,9 @@ import (
 
 //CreatePlan creates the plan from all planners
 func CreatePlan(ctx context.Context, inputPath, outputPath string, customizationsPath, transformerSelector, prjName string) plantypes.Plan {
+	if inputPath == "" && customizationsPath == "" {
```
Comment (ashokponkumar) [inline lib/planner.go:31]: Why should it fail when both of these are empty? Even internal transformers can be invoked by default, isn't it

**Review by ashokponkumar — CHANGES_REQUESTED** [review]

File: lib/planner.go (hunk 1 above)
Comment (kmehant) [reply lib/planner.go:31]: Ah, you are right!

File: lib/planner.go [hunk 2]
```diff
@@ -60,6 +60,10 @@ func CreatePlan(ctx context.Context, inputPath, outputPath string, customization
 	}
 	logrus.Infoln("Configuration loading done")
 
+	if len(p.Spec.Services) == 0 && len(p.Spec.InvokedByDefaultTransformers) == 0 {
```
Comment (ashokponkumar) [inline lib/planner.go:63]: Won't the list of services only become available only after the planning is done below?

**Review by ashokponkumar — CHANGES_REQUESTED** [review]

File: lib/planner.go (hunk 2 above)
Comment (kmehant) [reply lib/planner.go]: fixed it 😄

**Review by HarikrishnanBalagopal — APPROVED** [review]
LGTM

File: cmd/plan.go [hunk 3]
```diff
@@ -140,6 +140,10 @@ func planHandler(cmd *cobra.Command, flags planFlags) {
 		return
 	}
 	logrus.Infof("Plan can be found at [%s].", planfile)
+	if len(p.Spec.Services) == 0 && len(p.Spec.InvokedByDefaultTransformers) == 0 {
+		logrus.Debugf("Plan : %+v", p)
+		logrus.Fatalf("Failed to find any services or default transformers. Aborting.")
```
Comment (ashokponkumar) [inline cmd/plan.go:145]: Do we need the Aborting word? Since it has actually finished the process.

**Review by ashokponkumar — CHANGES_REQUESTED** [review]

File: cmd/plan.go (hunk 3 above)
Comment (kmehant) [reply cmd/plan.go]: correct, as we are creating the plan file. Fixed it.

**Review by ashokponkumar — APPROVED** [review]

**Pull request merged**